2021-09-29 17:42:40,571 INFO    MainThread:68560 [wandb_setup.py:_flush():69] setting env: {}
2021-09-29 17:42:40,572 INFO    MainThread:68560 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-29 17:42:40,572 INFO    MainThread:68560 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20210929_174240-j31ecwtx/logs/debug.log
2021-09-29 17:42:40,572 INFO    MainThread:68560 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20210929_174240-j31ecwtx/logs/debug-internal.log
2021-09-29 17:42:40,572 INFO    MainThread:68560 [wandb_init.py:init():381] calling init triggers
2021-09-29 17:42:40,572 INFO    MainThread:68560 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-09-29 17:42:40,572 INFO    MainThread:68560 [wandb_init.py:init():430] starting backend
2021-09-29 17:42:40,572 INFO    MainThread:68560 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-29 17:42:40,580 INFO    MainThread:68560 [backend.py:ensure_launched():135] starting backend process...
2021-09-29 17:42:40,633 INFO    MainThread:68560 [backend.py:ensure_launched():140] started backend process with pid: 72453
2021-09-29 17:42:40,635 INFO    MainThread:68560 [wandb_init.py:init():435] backend started and connected
2021-09-29 17:42:40,638 INFO    MainThread:68560 [wandb_init.py:init():494] updated telemetry
2021-09-29 17:42:40,639 INFO    MainThread:68560 [wandb_init.py:init():517] communicating current version
2021-09-29 17:42:42,003 INFO    MainThread:68560 [wandb_init.py:init():522] got version response 
2021-09-29 17:42:42,003 INFO    MainThread:68560 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-29 17:42:42,160 INFO    MainThread:68560 [wandb_init.py:init():557] starting run threads in backend
2021-09-29 17:42:47,178 INFO    MainThread:68560 [wandb_run.py:_console_start():1605] atexit reg
2021-09-29 17:42:47,178 INFO    MainThread:68560 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-09-29 17:42:47,179 INFO    MainThread:68560 [wandb_run.py:_redirect():1484] Redirecting console.
2021-09-29 17:42:47,186 INFO    MainThread:68560 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-29 17:42:47,186 INFO    MainThread:68560 [wandb_init.py:init():582] run started, returning control to user process
2021-09-29 17:43:02,200 INFO    MainThread:68560 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-09-29 17:43:02,201 INFO    MainThread:68560 [wandb_run.py:_restore():1547] restore
2021-09-29 17:43:04,300 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-29 17:43:05,008 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-29 17:43:05,161 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-29 17:43:05,267 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 10895
}

2021-09-29 17:43:05,380 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 10895
}

2021-09-29 17:43:05,484 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10895
  total_bytes: 10895
}

2021-09-29 17:43:05,588 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10895
  total_bytes: 10895
}

2021-09-29 17:43:05,691 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10895
  total_bytes: 10895
}

2021-09-29 17:43:05,795 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10895
  total_bytes: 10895
}

2021-09-29 17:43:05,914 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10895
  total_bytes: 10895
}

2021-09-29 17:43:06,224 INFO    MainThread:68560 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10895
  total_bytes: 10895
}
local_info {
}

2021-09-29 17:43:07,920 INFO    MainThread:68560 [wandb_run.py:_show_files():1947] logging synced files
